Example/ios/Podfile

```diff
@@ -2,12 +2,15 @@ require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 platform :ios, '11.0'
+install! 'cocoapods', :deterministic_uuids => false
 
 config = use_native_modules!
 use_react_native!(
   :path => config[:reactNativePath],
   # to enable hermes on iOS, change `false` to `true` and then install pods
-  :hermes_enabled => false
+  :hermes_enabled => false,
+  :fabric_enabled => false,
```

Comment:
I've noticed that Podfile of 0.68.0-rc.1 template uses `flags` dict here (see https://github.com/facebook/react-native/blob/v0.68.0-rc.1/template/ios/Podfile#L16). Can we copy the whole Podfile from react-native repo to eliminate all differences?